### DISCUSSION TITLE
Different error with Docker compose

### DISCUSSION THREAD

**Discussion by profnormano** [wgu-opensource/osmt#92] [Asking for Help] [answered]
Step 34/36 : RUN bash -c 'source $HOME/.nvm/nvm.sh   &&     nvm install node                    &&     npm install -g doctoc urchin eclint dockerfile_lint &&     npm install --prefix "$HOME/.nvm/"'
 ---> Running in 3c313ce86630
bash: /home/nvm/.nvm/nvm.sh: No such file or directory
1 error occurred:
        * Status: The command '/bin/bash -o pipefail -c bash -c 'source $HOME/.nvm/nvm.sh   &&     nvm install node                    &&     npm install -g doctoc urchin eclint dockerfile_lint &&     npm install --prefix "$HOME/.nvm/"'' returned a non-zero code: 1, Code: 1S

Suggestions welcome.

**Comment by JohnKallies** [comment, marked as answer]
Hello Norman. See my notes and request here: https://github.com/wgu-opensource/osmt/discussions/91. If there is additional follow up, let's have it in that thread? Thanks!
